src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -242,6 +243,25 @@ def GetSelectedIndex(self) -> "int":
         """
         pass
 
+    def OnDropDownOptionSelected(self, onSelectionChanged: "Any") -> "PyControlDropDown":
```

Ruff (0.14.8)

Undefined name `Any`

(F821)